src/lib/schemaparser.ts

```diff
@@ -0,0 +1,87 @@
+import {buildSchema} from 'graphql'
+import schemaExtras from './schemaextras'
+import * as difference from 'lodash/difference'
```

Comment:
import { difference } from 'lodash' pls

Reply:
```TypeError: difference_1.difference is not a function```
That does not work. Also this will not bloat space as it only imports from loadash/difference directly.